app/routes/_layout+/assets.$assetId.alerts.tsx

```diff
@@ -0,0 +1,230 @@
+    );
+  } catch (cause) {
+    const reason = makeShelfError(cause, { userId, assetId });
+    throw json(error(reason), { status: reason.status });
+  }
+}
+
+export async function action({ context, request }: ActionFunctionArgs) {
+  const authSession = context.getSession();
+  const userId = authSession.userId;
+
+  try {
+    const formData = await request.formData();
+
+    const { intent } = parseData(
+      formData,
+      z.object({ intent: z.enum(["edit-reminder", "delete-reminder"]) })
+    );
+
+    const { organizationId } = await requirePermission({
+      userId,
+      request,
+      entity: PermissionEntity.asset,
+      action: PermissionAction.update,
+    });
+
+    switch (intent) {
+      case "edit-reminder": {
+        const payload = parseData(
+          formData,
+          setReminderSchema.extend({ id: z.string() })
+        );
+
+        await editAssetReminder({
+          id: payload.id,
+          name: payload.name,
+          message: payload.message,
+          alertDateTime: payload.alertDateTime,
+          teamMembers: payload.teamMembers,
+          organizationId,
+        });
+
+        sendNotification({
+          title: "Reminder updated",
+          message: "Your asset reminder has been updated successfully",
+          icon: { name: "success", variant: "success" },
+          senderId: authSession.userId,
+        });
+
+        return json(data({ success: true }));
+      }
+
+      case "delete-reminder": {
+        const { id } = parseData(formData, z.object({ id: z.string().min(1) }));
+
+        await deleteAssetReminder({ id, organizationId });
+
+        sendNotification({
+          title: "Reminder deleted",
+          message: "Your asset reminder has been deleted successfully",
+          icon: { name: "trash", variant: "error" },
+          senderId: authSession.userId,
+        });
+
+        return json(data({ success: true }));
+      }
+
+      default: {
+        checkExhaustiveSwitch(intent);
+        return json(data(null));
+      }
+    }
+  } catch (cause) {
+    const reason = makeShelfError(cause, { userId });
+    return json(error(reason), { status: reason.status });
+  }
+}
+
+export default function AssetAlerts() {
+  return (
+    <List
+      className="overflow-x-visible md:overflow-x-auto"
+      ItemComponent={ListContent}
+      headerChildren={
+        <>
+          <Th>Message</Th>
+          <Th>Alert Date</Th>
+          <Th>Status</Th>
+          <Th>Users</Th>
+        </>
+      }
+    />
+  );
+}
+
+function ListContent({
+  item,
+}: {
+  item: Prisma.AssetReminderGetPayload<{
+    include: typeof ASSET_REMINDER_INCLUDE_FIELDS;
+  }> & { displayDate: string };
+}) {
+  const now = new Date();
+  const status =
+    now < new Date(item.alertDateTime) ? "Pending" : "Reminder sent";
+
+  return (
+    <>
+      <Td>{item.name}</Td>
+      <Td className="max-w-62 md:max-w-96">{item.message}</Td>
+      <Td>{item.displayDate}</Td>
+      <Td>{status}</Td>
```

Comment:
Lets use a pill like our other statuses. Lets do green for sent and yellow for pending.